programs/drift/src/controller/orders.rs

```diff
@@ -1025,6 +1025,16 @@ pub fn fill_perp_order(
         return Ok(0);
     }
 
+    {
```

Comment:
Do we needa do this for spot fills too?

Reply:
dont have an amm to throw the same error ... i guess could just change it so there is no payment for canceling limit order thats expired